README.md

```diff
@@ -14,6 +14,21 @@ The key difference between [UiAutomator2 Driver](https://github.com/appium/appiu
 
 * If there are ever problems starting a session, try setting the capability `forceEspressoRebuild=true` and retrying. This will rebuild a fresh Espresso Server APK. If the session is succcesful, set it back to false so that it doesn't re-install on every single test.
 * Espresso requires the debug APK and app-under-test APK (AUT) to have the same signature. It automatically signs the AUT with the `io.appium.espressoserver.test` signature. This may have problems if you're using an outdated Android SDK tools and/or an outdated Java version.
+* If you failed to establish Espresso session in `proguard` environment because of `Resources$NotFoundException`, for example, please try to add below rule. Please read [#449](https://github.com/appium/appium-espresso-driver/issues/449#issuecomment-537833139) for more various cases.
```

Comment:
If you experience session startup failures due to exceptions similar to `Resources$NotFoundException` then try to adjust your Proguard rules:

....

Please read [#449](https://github.com/appium/appium-espresso-driver/issues/449) for more details on this topic